langstream-cli/src/main/java/ai/langstream/cli/commands/python/LoadPythonDependenciesCmd.java

```diff
@@ -0,0 +1,213 @@
+/*
+ * Copyright DataStax, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package ai.langstream.cli.commands.python;
+
+import ai.langstream.cli.commands.VersionProvider;
+import java.io.File;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.time.Duration;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.Executors;
+import java.util.concurrent.atomic.AtomicReference;
+import lombok.SneakyThrows;
+import org.apache.commons.io.input.Tailer;
+import org.apache.commons.io.input.TailerListener;
+import picocli.CommandLine;
+
+@CommandLine.Command(
+        name = "load-pip-requirements",
+        header = "Process python dependencies in requirements.txt")
+public class LoadPythonDependenciesCmd extends BasePythonCmd {
+
+    private static final AtomicReference<ProcessHandle> dockerProcess = new AtomicReference<>();
+
+    @CommandLine.Option(
+            names = {"-app", "--application"},
+            description = "Application directory path",
+            required = true)
+    private String appPath;
+
+    @CommandLine.Option(
+            names = {"--docker-args"},
+            description = "Additional docker arguments")
+    private List<String> dockerAdditionalArgs = new ArrayList<>();
+
+    @CommandLine.Option(
+            names = {"--docker-command"},
+            description = "Command to run docker")
+    private String dockerCommand = "docker";
+
+    @CommandLine.Option(
+            names = {"--langstream-runtime-version"},
+            description = "Version of the LangStream runtime to use")
+    private String dockerImageVersion = VersionProvider.getMavenVersion();
+
+    @CommandLine.Option(
+            names = {"--langstream-runtime-docker-image"},
+            description = "Docker image of the LangStream runtime to use")
+    private String dockerImageName;
+
+    @Override
+    @SneakyThrows
+    public void run() {
+
+        if (dockerImageVersion != null && dockerImageVersion.endsWith("-SNAPSHOT")) {
```

Comment:
can we move it to a common function? is the same logic of docker run, right ?

Reply:
good point, done